tapdb/assets_store.go

```diff
@@ -1819,9 +1819,13 @@ func (a *AssetStore) queryCommitments(ctx context.Context,
 
 // LogPendingParcel marks an outbound parcel as pending on disk. This commits
 // the set of changes to disk (the pending inputs and outputs) but doesn't mark
-// the batched spend as being finalized.
+// the batched spend as being finalized. The final lease owner and expiry are
+// the lease parameters that are set on the input UTXOs, since we assume the
+// parcel will be broadcast after this call. So we'll want to lock the input
+// UTXOs for forever, which means the expiry should be far in the future.
 func (a *AssetStore) LogPendingParcel(ctx context.Context,
-	spend *tapfreighter.OutboundParcel) error {
+	spend *tapfreighter.OutboundParcel, finalLeaseOwner [32]byte,
+	finalLeaseExpiry time.Time) error {
```

Comment:
nit: I don't understand why we need the `final` prefix here in `finalLeaseOwner` and `finalLeaseExpiry`. Consider removing?